### PR TITLE
3.4 - Remove more deprecated function's call

### PR DESCRIPTION
**Pull request by antograssiot**
I found those mainly related to events and adjusted some docblocks

### CODE REVIEW

File: src/TestSuite/Constraint/EventFiredWith.php [hunk 1]
```diff
@@ -83,14 +83,14 @@ public function matches($other)
         if (count($events) > 1) {
             throw new PHPUnit_Framework_AssertionFailedError(sprintf('Event "%s" was fired %d times, cannot make data assertion', $other, count($events)));
         }
-
+        /** @var \Cake\Event\Event $event */
```
Comment (antograssiot) [inline src/TestSuite/Constraint/EventFiredWith.php]: well it is bot 😛

Comment (dereuromark) [inline src/TestSuite/Constraint/EventFiredWith.php]: I think what he means is /* to use :) Until we can fix up the sniffer here (I got one working, but someone needs to migrate it to CakePHP)

Comment (antograssiot) [inline src/TestSuite/Constraint/EventFiredWith.php]: @dereuromark is your sniff v3 ready now ?

Comment (dereuromark) [inline src/TestSuite/Constraint/EventFiredWith.php]: With minimal adjustments probably. Slack me and I can forward the infos.

**Comment by markstory** [comment]
Great work @antograssiot !

**Pull request merged**